tests/tests_test_workflow/test_integ_workflow/integ_test/data/build_manifest_missing_components.yml

```diff
@@ -2,6 +2,7 @@
 build:
   platform: linux
   architecture: x64
+  distribution: None
```

Comment:
Add a 1.3.0 manifest that has a value for distribution. Possibly a new test.

`distribution: None` is not a thing, I don't expect to ever find this in a build file, when we don't pass distribution it defaults to something (tar?) doesn't it?

Reply:
I was following the suggestion to have `None` as default, while previously my commit I use `tar` as default.
I am bit confused here what exactly should I use?
Thanks.